src/zenml/cli/server.py

```diff
@@ -55,60 +61,68 @@ def explain_server() -> None:
 
 
 @server.command("up", help="Start a daemon service running the ZenServer.")
+@click.option(
+    "--ip-address", type=IP_ADDRESS, default="127.0.0.1", show_default=True
+)
 @click.option("--port", type=int, default=8000, show_default=True)
 @click.option("--profile", type=str, default=None)
-def up_server(port: int, profile: Optional[str]) -> None:
+def up_server(
+    ip_address: Union[ipaddress.IPv4Address, ipaddress.IPv6Address],
+    port: int,
+    profile: Optional[str],
+) -> None:
     """Provisions resources for the ZenServer."""
     from zenml.services import ServiceRegistry
     from zenml.zen_server.zen_server import ZenServer, ZenServerConfig
 
+    print(type(ip_address))
```

Comment:
One `print` gone, another one appeared 😄

Reply:
OMG, this is embarassing :)